kolibri/utils/server.py

```diff
@@ -65,6 +65,9 @@ def start(port=8080):
     # start the pingback thread
     PingbackThread.start_command()
 
+    # Do a db vacuum periodically
+    VacuumThread.start_command()
```

Comment:
Unconfigurable thread behavior breaks the intention of having this under `cron` control.

I'm a bit confused at this point, because there are 3 approaches:

1) Iceqube (which IMO isn't perfect)
2) Cron jobs (of course .deb pkg only)
3) A basic Python thread

Reply:
We don't have the intention of running it under cron as it's not possible in Windows/pex installations. It's done using point 3 of your list . It seems to be the only possible universal solution.